apb.yml

```diff
@@ -1,5 +1,5 @@
 ---
-version: 1.1
+version: 1.2
```

Comment:
Sorry I missed this in your last PR, this should actually stay `1.0`. This refers to the apb.yml spec version which is still at 1.0.